templates/misc/zope_urls_from_db.py

```diff
@@ -0,0 +1,22 @@
+import psycopg2
+
+
+connection_string = (
+    'postgres://{{ archive_db_user }}:{{ archive_db_password }}'
```

Comment:
Yes, we actually have the database details in zope so I can just use `./bin/instance run`.